src/main/res/values/strings.xml

```diff
@@ -157,7 +157,7 @@ You should have received a copy of the GNU General Public License along with Tod
     <string name="share_task_show_edit_summary">Edit tasks shared to Simpletask before saving them</string>
 
     <string name="share_task_append_text_title">Background text</string>
-    <string name="share_task_append_text_summary">Append \"%s\" to tasks added in the background. Set a list or tag to allow easy filtering to view these tasks.</string>
+    <string name="share_task_append_text_summary">Append \"%s\" to tasks added in the background.</string>
```

Comment:
I'd like to avoid scrolling in preference categories if we can help it. Making this description shorter makes it only take one line, which makes us closer to having no scrolling in 'other'.